### DISCUSSION TITLE
Wireguard UI Script

### DISCUSSION THREAD

**Discussion by yodaphone** [tteck/Proxmox#820] [Ideas]
A script for this will be really helpful 

https://github.com/ngoduykhanh/wireguard-ui

A wireguard UI container built on docker & also a binary. 

Is this something you will consider to create a script for?

**Comment by tteck** [comment]
The script currently uses WGDashboard, is WireGuard-UI better?

**Comment by yodaphone** [comment]
will give this a try, didnt realize you have a UI version

**Comment by yodaphone** [comment]
tried your build but i personally feel the one i suggested has more features and easier to manage. for e.g. PreUp and PostDown etc.

**Reply by tteck** [reply]
I'll look into
